test/events/ApplyDamageTest.cpp

```diff
@@ -22,9 +22,11 @@ namespace test
 using namespace ::testing;
 using namespace ::events;
 
-class ListenerMock
+class ApplyDamageListenerMock
 {
 public:
+	virtual ~ApplyDamageListenerMock() = default;
```

Comment:
not quite clear what's the purpose of the virtual destructor if nobody inherits from this class